server/src/routes/v2/apps.js

```diff
@@ -64,4 +87,207 @@ module.exports = [
             })
         },
     },
+    {
+        method: 'POST',
+        path: '/v2/apps',
+        config: {
+            auth: 'token',
+            tags: ['api', 'v2'],
+            payload: {
+                maxBytes: 20 * 1024 * 1024, //20MB
+                allow: 'multipart/form-data',
+                parse: true,
+                output: 'stream',
+                multipart: true,
+            },
+            validate: {
+                payload: CreateAppModel.payloadSchema,
+            },
+            plugins: {
+                'hapi-swagger': {
+                    payloadType: 'form',
+                },
+            },
+        },
+        handler: async (request, h) => {
```

Comment:
I think we should extract some of these to an Apps-Service, and structure it more like organisation routes. Handler is still too complex imo